sources/src/configuration.ts

```diff
@@ -216,6 +216,46 @@ export class BuildScanConfig {
         return core.getInput('develocity-token-expiry')
     }
 
+    getDevelocityInjectionEnabled(): string {
```

Comment:
This should be a boolean value, and use `getBooleanInput()`

Reply:
added a `getMaybeBooleanInput` that returns a `boolean | undefined`  https://github.com/gradle/actions/pull/244/commits/f616d16c338b744d5d1c252fd08733955caf2090